### ISSUE TITLE
Error during Cuckoo analysis

### ISSUE THREAD

**Issue by akosp94** [cuckoosandbox/cuckoo#1984]
Okay so I have a diploma work in progress and I use Cuckoo on 16.04 LTS ubuntu, my guest vm is running on the latest virtualbox version, with Windows 7 x64 sp1. I did my configuration, I am able to ping one another from both machines, yet I get this: 
 Checking for updates...
 You're good to go!
2017-11-28 09:18:47,666 [cuckoo.core.scheduler] INFO: Using "virtualbox" as machine manager
2017-11-28 09:18:48,353 [cuckoo.core.scheduler] INFO: Loaded 1 machine/s
2017-11-28 09:18:48,368 [cuckoo.core.scheduler] INFO: Waiting for analysis tasks.
2017-11-28 09:19:05,771 [cuckoo.core.scheduler] INFO: Starting analysis of FILE "Mesterséges_intelligencia.doc" (task #17, options "procmemdump=yes,route=none")
2017-11-28 09:19:05,854 [cuckoo.core.scheduler] INFO: Task #17: acquired machine cuckoo1 (label=win7x64_szakdoga)
2017-11-28 09:19:05,862 [cuckoo.auxiliary.mitm] INFO: Started mitm interception with PID 9915 (ip=0.0.0.0, port=50000).
2017-11-28 09:19:05,871 [cuckoo.auxiliary.sniffer] INFO: Started sniffer with PID 9916 (interface=vboxnet0, host=192.168.56.101)
2017-11-28 09:19:09,222 [cuckoo.core.guest] INFO: Starting analysis on guest (id=cuckoo1, ip=192.168.56.101)
2017-11-28 09:19:17,061 [cuckoo.core.guest] INFO: Guest is running Cuckoo Agent 0.7 (id=cuckoo1, ip=192.168.56.101)
2017-11-28 09:22:51,782 [cuckoo.core.guest] INFO: cuckoo1: end of analysis reached!
2017-11-28 09:22:54,700 [cuckoo.machinery.virtualbox] INFO: Successfully generated memory dump for virtual machine with label win7x64_szakdoga to path /home/thewantedgun/.cuckoo/storage/analyses/17/memory.dmp
2017-11-28 09:23:06,992 [cuckoo.processing.behavior] WARNING: Analysis results folder does not exist at path '/home/thewantedgun/.cuckoo/storage/analyses/17/logs'.
[x64] Gathering all referenced SSDTs from KeAddSystemServiceTable...
Finding appropriate address space for tables...
2017-11-28 09:24:51,836 [cuckoo.processing.debug] ERROR: Error processing task #17: it appears that the Virtual Machine hasn't been able to contact back to the Cuckoo Host. There could be a few reasons for this, please refer to our documentation on the matter: https://cuckoo.sh/docs/faq/index.html#troubleshooting-vm-network-configuration
2017-11-28 09:24:52,516 [cuckoo.core.plugins] ERROR: Failed to run the reporting module: SingleFile
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/cuckoo/core/plugins.py", line 614, in process
    current.run(self.results)
  File "/usr/local/lib/python2.7/dist-packages/cuckoo/reporting/singlefile.py", line 106, in run
    f = weasyprint.HTML(io.BytesIO(report.encode("utf8")))
  File "/usr/local/lib/python2.7/dist-packages/weasyprint/__init__.py", line 95, in __init__
    namespaceHTMLElements=False)
TypeError: parse() got an unexpected keyword argument 'override_encoding'
2017-11-28 09:24:52,785 [cuckoo.core.scheduler] INFO: Task #17: reports generation completed
2017-11-28 09:24:52,888 [cuckoo.core.scheduler] INFO: Task #17: analysis procedure completed

So basically something is off with the networking, even though I use Host-only networking with a virtual interface named vboxnet0 using 192.168.56.1 as default IP, my VM has 192.168.56.101 as IP, ...56.1 as Default Gateway.

**Comment by akosp94** [comment]
*note, I'm a noob in Linux, so please answer thoroughly! :smile:

**Comment by akosp94** [comment]
Got it all figured out, the result server IP was wrong as well as the port.
